pulsar4s-akka-streams/src/main/scala/com/sksamuel/pulsar4s/akka/streams/package.scala

```diff
@@ -32,9 +33,10 @@ package object streams {
    */
   def committableSource[T](
     create: () => Consumer[T],
-    seek: Option[MessageId] = None
+    seek: Option[MessageId] = None,
+    closeDelay: FiniteDuration = 30.seconds,
```

Comment:
This breaks binary compatibility, so we could add an extra overload to fix that, or we can avoid changing the API altogether here and only change it in `PulsarCommittableSourceGraphStage`. It should be fine if we pick a reasonable/generous default for `closeDelay`.

Reply:
I think its worth adding the overload and deprecating it if necessary.